pkg/tfgen/docs.go

```diff
@@ -594,21 +599,34 @@ func convertHCL(hcl string) (string, string, error) {
 	return result, "", nil
 }
 
-func cleanupDoc(g *generator, info tfbridge.ResourceOrDataSourceInfo, doc parsedDoc) parsedDoc {
+func cleanupDoc(g *generator, info tfbridge.ResourceOrDataSourceInfo, doc parsedDoc) (parsedDoc, error) {
 	newargs := make(map[string]string, len(doc.Arguments))
 	for k, v := range doc.Arguments {
-		newargs[k] = cleanupText(g, info, v)
+		cleanupText, err := cleanupText(g, info, v)
+		if err != nil {
+			return parsedDoc{}, err
```

Comment:
Is it correct to bail early and return an empty parseddoc?  Don’t we want to still compute this as normal and just pass back a bool indicating a that it contained elided?

We could even likely skip most of what’s added here and just compare the result the result of cleanupText with elidedDocComment in the places we want to check if we should warn?